src/iuse.cpp

```diff
@@ -7311,7 +7318,7 @@ static extended_photo_def photo_def_for_camera_point( const tripoint &aim_point,
     // TODO: fix point types
     const oter_id &cur_ter =
         overmap_buffer.ter( tripoint_abs_omt( ms_to_omt_copy( here.getabs( aim_point ) ) ) );
-    std::string overmap_desc = string_format( _( "In the background you can see a %s" ),
+    std::string overmap_desc = string_format( _( "In the background you can see a %s." ),
```

Comment:
This string isn't supposed to have a period. It may be concatenated with " interior" at line 7327/7331, and will then be concatenated with a period at line 7336.

Reply:
Oh, bugger. Well, dammit. Now what?

Reply:
Well, fix it maybe? I'll PR a fix later if you do not.

Reply:
This PR has already been merged. If I make another commit, what happens?

Reply:
It won't be added to this PR, but you can make a new PR.